app/controllers/loadBasicPokemonDetails.js

```diff
@@ -8,20 +8,20 @@ function loadBasicPokemonDetails() {
     console.log('Loading Basic Pokemon Details to MongoDB');
 
     let data = JSON.stringify(pokemonListPath, function(key,value) {
-        var pokemonID, pokemonName, male, female;
+        var pokemonID, pokemonName, male;
         if(key != 'name' && typeof value.name !== "undefined") {
             pokemonID = key;
             pokemonName = value.name;
             male = value.gender.male;
-            female = value.gender.female;
+            //female = value.gender.female;
```

Comment:
Another suggestion: git keeps the history of files. Commenting out old code is not a good practice, just delete it, you have revisions! :)